api-compact.yaml

```diff
@@ -62,9 +65,40 @@ apigateway_responses_with_mapping_template: &apigateway_responses_with_mapping_t
   500:
     statusCode: "500"
 
-produces: &produces
-  - "application/json"
-  - "application/csv"
+apigateway_responses_with_mapping_template_for_uuid: &apigateway_responses_with_mapping_template_for_uuid
```

Comment:
@saleem-latif Unfortunately, this one is being used for the `enterprise_catalogs/{uuid}` endpoint, too :\